application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -64,69 +64,65 @@
  *   copy-on-write and shares a lot of objects with any other TransitLayer instances.
  */
 public interface TransitService {
-  Collection<String> getFeedIds();
+  Collection<String> listFeedIds();
 
-  Collection<Agency> getAgencies();
-  Optional<Agency> findAgencyById(FeedScopedId id);
+  Collection<Agency> listAgencies();
+  Optional<Agency> findAgency(FeedScopedId id);
 
   FeedInfo getFeedInfo(String feedId);
 
-  Collection<Notice> getNoticesByEntity(AbstractTransitEntity<?, ?> entity);
+  Collection<Notice> findNotices(AbstractTransitEntity<?, ?> entity);
```

Comment:
This is a bit on the edge if we can use the standard `findX` convention for this method. In general, I think there are a lot of cases where it would improve the readability a bit to explains in the method name how the argument is used for the search. I would prefer `findNoticesForEntity` in this case. We can discuss this in a dev meeting.